web/settings.py

```diff
@@ -73,8 +73,17 @@ class Common(Configuration):
 
     WSGI_APPLICATION = "web.wsgi.application"
 
-    default_db = "postgres://postgres:postgres@db:5432/mariaquiteria"
-    DATABASES = {"default": dj_database_url.config(default=default_db)}
+    # Precisamos alterar para variavel de ambiente: default_db = postgres://POSTGRES_USER:$POSTGRES_PASSWORD@POSTGRES_ENDPOINT:5432/POSTGRES_NAME
+    #default_db = "postgres://postgres:postgres@mariaquiteria.c4rmfbajkjko.us-east-1.rds.amazonaws.com/mariaquiteria"
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'POSTGRES_ENDPOINT': os.getenv('POSTGRES_ENDPOINT'),
+            'POSTGRES_NAME': os.getenv('POSTGRES_NAME'),
+            'POSTGRES_USER': os.getenv('POSTGRES_USER'),
+            'POSTGRES_PASSWORD': os.getenv('POSTGRES_PASSWORD'),
+        }
+    }
```

Comment:
Não sei se essa é maneira mais elegante para utilizar variaveis de ambiente. 
```suggestion
    DATABASES = {
        'default': {
            'ENGINE': 'django.db.backends.postgresql',
            'POSTGRES_ENDPOINT': os.getenv('POSTGRES_ENDPOINT'),
            'POSTGRES_NAME': os.getenv('POSTGRES_NAME'),
            'POSTGRES_USER': os.getenv('POSTGRES_USER'),
            'POSTGRES_PASSWORD': os.getenv('POSTGRES_PASSWORD'),
        }
    }
```

Reply:
Tá OK pra mim isso :+1: 

Uma outra opção seria usar o [`dj-database-url`](https://pypi.org/project/dj-database-url/) para parsear uma variável `DATABASE_URL` que é geralmente usada em serviços, mas não é necessário.

Reply:
Acabei de notar que o `dj-database-url`[já está sendo utilizado em prod](https://github.com/DadosAbertosDeFeira/maria-quiteria/blob/9c99062c8a69309784af7236d1bdec36e42484c9/web/settings.py#L174), então eu acho que só precisa definir a variável de ambiente `DATABASE_URL`.